docs/change-log/index.md

```diff
@@ -310,25 +483,33 @@ with a drill-down to view the steps and their details. [Tech Preview]
-| 2407 | Kafka ingestion service on empty feature set returns an error. | Ingest a sample of the data manually. This creates the schema for the feature set and then the ingestion service accepts new records. | 1.1.0 |
+| 2223 | Cannot deploy a function when notebook names contain "." (ModuleNotFoundError) | Do not use "." in notebook name | v1.0.0  |
+| 2199 | Spark operator job fails with default requests args.       | NA                                         | v1.0.0 |
+| 1584 | Cannot run `code_to_function` when filename contains special characters | Do not use special characters in filenames | v1.0.0 |
+| [2621](https://github.com/mlrun/mlrun/issues/2621) | Running a workflow whose project has `init_git=True`, results in Project error | Run `git config --global --add safe.directory '*'` (can substitute specific directory for *). | v1.1.0 |
+| 2407 | Kafka ingestion service on sn empty feature set returns an error. | Ingest a sample of the data manually. This creates the schema for the feature set and then the ingestion service accepts new records. | v1.1.0 |
+|  | The feature store does not support schema evolution and does not have schema enforcement. | NA | v1.2.1 |
+| 3420 | MLRun database doesn't raise an exception when the blob size is greater than 16,777,215 bytes | NA      | v1.2.1 |
+| 3386 | Documentation is missing full details on the feature store sources and targets | NA | v1.2.1 |
+| 2421 | Artifact logged via SDK with "/" in the name cannot be viewed in the UI. The main project dashboard opens instead. | NA | v1.1.0 |
+| 3424 | Documentation missing a matrix of which engines support which sources/targets | NA                        | v1.2.1 |
 
+    
+    
 ## Limitations
 
 
 | ID   | Description                                                    | Workaround                           | Opened | 
 | ---- | -------------------------------------------------------------- | ------------------------------------ | ----------|      
-| 2014 | Model deployment returns ResourceNotFoundException (Nuclio error that Service <name> is invalid.) | Verify that all `metadata.labels` values are 63 characters or less (Kubernetes limitation). |  v1.0.0  |
-
+| 2014 | Model deployment returns ResourceNotFoundException (Nuclio error that Service <name> is invalid.) | Verify that all `metadata.labels` values are 63 characters or less. See the [Kubernetes limitation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set). |  v1.0.0  |
+| 3315 | The feature store does not support an aggregation of aggregations | NA                                  | v1.2.1 |
+| 3381 | Private repo is not supported as a marketplace hub             | NA                                     | 1.2.1 | 
```

Comment:
remove `v` / use `v` as prefix from `version` for consistency. see `1.2.1` vs `v1.2.1`

Reply:
fixed